lib/lightning/usage_tracking/run_service.ex

```diff
@@ -15,11 +15,11 @@ defmodule Lightning.UsageTracking.RunService do
     |> finished_on(date)
   end
 
-  def unique_jobs(steps, date) do
+  def unique_job_ids(steps, date) do
     steps
     |> finished_on(date)
-    |> Enum.map(& &1.job)
-    |> Enum.uniq_by(& &1.id)
+    |> Enum.map(& &1.job.id)
```

Comment:
I think this will have the same effect, if the job doesn’t exist the job assoc will be nil, you can get `job_id` off of the step. Am I not understanding this?

What constitutes active in this case?

Reply:
@stuartc Well caught - that was a typo - corrected.